### DISCUSSION TITLE
Datasets

### DISCUSSION THREAD

**Discussion by amberwnaushahi** [ALIYA2Group/Mod20_Segment_1#14] [Ideas]
Trying to explore datasets we can use for our analysis. The way I see it, we need:
1) Temperature data for Arctic and Antarctic (date/time/location(Long_Lat to link to other datasets)
2) Sea Ice Data for Arctic and Antarctic (incl various variable such as wind pressure , thickness etc)
3) Atmospheric Pressure
4) Animal population and extinction data for Arctic and Antarctic

**Comment by amberwnaushahi** [comment]
1- use historical sea ice data for Arctic and Antarctic (by date) - NSIDC
2- retrieve temperature and climate parameters by date for Arctic and Antarctic - (the sea ice data is a mean, based over a large area - we have to decide if we will take this data for a specific location such as the Arctic Basin, or retrieve for various locations and average them out during the data transformation process)
3- combine sea ice data with temperature and climate data (1 +2 ) into 1 dataset
4-  use dataset in 3 and run machine learning model with sea ice as target and rest of parameters as X (weights assigned through the model) to predict Sea Ice diminishing
5- retrieve/scrape live weather data for 2) to predict sea ice - 
6 - retrieve the wildlife endangered list for Arctic and predict extinction based on Sea Ice and Temperatures - 
7 - retrieve the wildlife endangered list for Antarctic and predict extinction based on Sea Ice and Temperatures - 

We will have 4 models:

1) Arctic
2) Antarctic
3) Wildlife Extinction for Arctic based on the sea ice diminish
4) Wildlife Extinction for Antarctic based on the sea ice diminish

We can also restrict the analysis to only Arctic

**Reply by amberwnaushahi** [reply]
I have noticed in my Data Search and Aryam would probably agree that while this topic is awesome, the data for this is quite hard to find and not consistent. It will either require lots of cleaning and in the end might not even make sense. 

@maryamt95 what do you think?

The Arctic sea ice is spread over a wide area and thus the factors impacting it would include multiple factors including multi-location temperatures ranging from Alaska, Greenland to Canada and Russia.

**Comment by seafishleo** [comment]
Got a question, that when we confirm our working dataset (whether it's a json or pandas data form), we use these numeric data for ML prediction. But how do we export/transform those data into a geospatial data that can be applied on the Google earth? (not a bar chart or line chart)
